src/main/java/frc/robot/commands/FeederIndexCommand.java

```diff
@@ -36,13 +36,13 @@ public void execute() {
         }
         // If there is a ball completely within conveyance stage 2,
         // set isBallBetweenSensors to false and stop conveyance stage 2
-        if(Robot.FEEDER_SUBSYSTEM.getFeederSubsystemHasBall() == true) {
+        if(Robot.FEEDER_SUBSYSTEM.getFeederHasBall() == true) {
```

Comment:
Use "else if" here, its easier to reason that the first matching condition is what is acted on. In this case though, we act on every matching condition.